README.adoc

```diff
@@ -34,6 +34,7 @@ buildPlugin(
 * `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
 * `useContainerAgent` (default: `false`) - uses a link:https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#container-agents[Container agent] instead of a Virtual Machine: usually faster to start and generates less costs for the project
 ** Please note that the implementation of "containers" can be changed over time
+* `forkCount` (default: null) - Execute tests with `forkCount` Java virtual machines. Common values include `0.45C`, `1`, `2`, and `1C` (if you terminate the value with a 'C', that value will be multiplied by the number of available CPU cores).
```

Comment:
May want to explain that this controls the number of JVMs used for the test code itself. That will also include code under test in the case of unit tests and `JenkinsRule`, but additional JVMs may be launched if you attach agents or use `RealJenkinsRule`, and especially in those cases setting this too high can lead to memory problems.

Reply:
Done in 942490c939a853e94f38f43fcfe9df1776dfafc3

Reply:
The explanation is mangled I think. The fork count should be the (maximum) number of concurrent JVMs running test code and any same-process code under test, such as controller code when using `JenkinsRule`. There will be _additional_ JVMs if you start agents or use `RealJenkinsRule`.

Reply:
Thanks.  I had misunderstood.  Corrected in fe9fb49f869230697992caf599e3de4e4b02313a

Reply:
Strange that my local git client reports that the commit fe9fb49 has been pushed, but the GitHub user interface does not show it as pushed. I can pull the change from another machine, but the UI does not show it...

Reply:
https://www.githubstatus.com/

![image](https://github.com/jenkins-infra/pipeline-library/assets/21194782/64143d8a-30e6-4282-bfd7-208f51036e91)